tests/test_agent_model_keys_route.py

```
@@ -0,0 +1,84 @@
+
+
+@pytest.fixture(autouse=True)
+def _ensure_agent_model_key_store(client, tmp_path_factory):
+    """Init app.state.agent_model_keys on a fresh DB; the test client registers
+    the store but does not run the lifespan that init()s it (production does)."""
+    store = client._transport.app.state.agent_model_keys
+    if store._db is not None:
+        try:
+            asyncio.get_event_loop().run_until_complete(store.close())
+        except Exception:
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)